src/web/app/Page.re

```diff
@@ -62,11 +66,27 @@ module Update = {
     | Globals(Globals.Update.t)
     | Editors(Editors.Update.t)
     | ExplainThis(ExplainThisUpdate.update)
+    | Assistant(AssistantUpdate.t)
     | MakeActive(selection)
     | Benchmark(benchmark_action)
     | Start
     | Save;
 
+  // Helper function to check for insertion of '??' or '?a' and trigger assistant updates
+  let send_assistant_insertion_info =
```

Comment:
this fn seems to be a fairly thin wrapper around check_req? is it worth defining?